benchmark/tpch.md

```diff
@@ -0,0 +1,90 @@
+
+## 测试环境
+
+### 1. 机器硬件信息
+
+| 类别       |  名称                                                |
+|------------|------------------------------------------------------|
+| OS         | linux (CentOS 7.3.1611)                              |
+| CPU        | 40 vCPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz  |
+| RAM        | 128GB                                                |
+| DISK       | 1.5T SSD * 2  + Optane SSD * 1                       |
+
+### 2. TPC-H
+
+[tidb-bench/tpch](https://github.com/pingcap/tidb-bench/tree/master/tpch)
+
+### 3. 集群拓扑
+
+| 机器 IP      | 部署的实例 |
+|--------------|------------|
+| 172.16.31.2  | TiKV \* 2  |
+| 172.16.31.3  | TiKV \* 2  |
+| 172.16.31.6  | TiKV \* 2  |
+| 172.16.31.8  | TiKV \* 2  |
+| 172.16.31.10 | TiKV \* 2  |
+| 172.16.31.10 | PD \* 1    |
+| 172.16.31.4  | TiDB \* 1  |
+
+### 4. 相应 TiDB 版本信息
+
+TiDB 1.0：
+
+| 组件名 | 版本号      | commit hash                                |
+|--------|-------------|--------------------------------------------|
+| TiDB   | v1.0.9      | 4c7ee3580cd0a69319b2c0c08abdc59900df7344   |
+| TiKV   | v1.0.8      | 2bb923a4cd23dbf68f0d16169fd526dc5c1a9f4a   |
+| PD     | v1.0.8      | 137fa734472a76c509fbfd9cb9bc6d0dc804a3b7   |
+
+TiDB 2.0：
+
+| 组件名 | 版本号      | commit hash                                |
+|--------|-------------|--------------------------------------------|
+| TiDB   | v2.0.0-rc.6 | 82d35f1b7f9047c478f4e1e82aa0002abc8107e7   |
+| TiKV   | v2.0.0-rc.6 | 8bd5c54966c6ef42578a27519bce4915c5b0c81f   |
+| PD     | v2.0.0-rc.6 | 9b824d288126173a61ce7d51a71fc4cb12360201   |
+
+## 测试结果
+
+| Query ID	| TiDB 2.0 	         | TiDB 1.0         |
+|-----------|--------------------|------------------|
+| 1	        | 33.9154210091s	 | 215.305725098s   |
+| 2	        | 25.5757238865s	 | Nan              |
+| 3	        | 59.6319231987s	 | 196.003419161s   |
+| 4	        | 30.2346801758s	 | 249.919836998s   |
+| 5	        | 31.6665229797s	 | OOM              |
+| 6	        | 13.1118650436s	 | 118.709993839s   |
+| 7	        | 31.7108299732s	 | OOM              |
+| 8	        | 31.7343668938s	 | 800.546444893s   |
+| 9	        | 34.2111070156s	 | 630.639750004s   |
+| 10	    | 30.7746679783s	 | 133.547474861s   |
+| 11	    | 27.6926951408s	 | 78.0269281864s   |
+| 12	    | 27.9628179073s	 | 124.641829014s   |
+| 13	    | 27.6760618687s	 | 174.695388079s   |
+| 14	    | 19.6767029762s	 | 110.602974892s   |
+| 15	    | NaN	             | Nan              |
+| 16	    | 24.8907389641s	 | 40.5292630196s   |
+| 17	    | 245.796802998s	 | NaN              |
+| 18	    | 91.2561578751s	 | OOM              |
+| 19	    | 37.6150860786s	 | NaN              |
+| 20	    | 44.1677880287s	 | 212.201659918s   |
+| 21	    | 31.4664649963s	 | OOM              |
+| 22	    | 31.5397689342s	 | 125.471378088s   |
+
+说明：
+1. Query 15 1.0 和 2.0 都还未支持视图，所以结果标记为 NaN
+2. Query 2, 17, 19 因为 TiDB 1.0 长时间未跑出结果，所以结果标记为 Nan
+3. Query 5, 7, 18, 21 因为 TiDB 1.0 在跑的过程中内存占用过多被 oom-killer 杀死，所以结果标记为 OOM
```

Comment:
The ordered list is usually used for steps. Use the bulleted list if the users do not need to perform in order.